packages/bff/src/services/authorizationService.ts

```diff
@@ -144,6 +180,40 @@ export function authorizationServiceBuilder(
         });
       return sessionToken;
     },
+    samlLoginCallback: async (
+      correlationId: string,
+      samlResponse: string
+    ): Promise<string> => {
+      validateSamlResponse(samlResponse);
+
+      const { serialized } =
+        await interopTokenGenerator.generateInternalToken();
+
+      const headers = {
+        "X-Correlation-Id": correlationId,
+        Authorization: `Bearer ${serialized}`,
+      };
+
+      const tenant = await tenantProcessClient.tenant.getTenant({
+        params: { id: config.pagoPaTenantId },
+        headers,
+      });
+
+      const selfcareId = tenant.selfcareId;
+      if (!selfcareId) {
+        throw missingSelfcareId(config.pagoPaTenantId);
+      }
+
+      const claims = buildSupportClaims(selfcareId, tenant);
+
+      const { serialized: sessionToken } =
+        await interopTokenGenerator.generateSessionToken(
+          claims,
+          config.supportLandingJwtDuration
```

Comment:
this endpoint uses `supportJwtDuration` config, not `supportLandingJwtDuration`.
May I ask to adapt env configurations accordingly?

Reply:
According to scala [code](https://github.com/pagopa/interop-be-backend-for-frontend/blob/1.0.x/src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AuthorizationApiServiceImpl.scala#L147), it uses `supportLandingJwtDuration` that it seems to be taken [from](https://github.com/pagopa/interop-be-backend-for-frontend/blob/1.0.x/src/main/scala/it/pagopa/interop/backendforfrontend/common/system/ApplicationConfiguration.scala#L25) `backend-for-frontend.saml2-jwt.support-landing-token-duration-seconds` that seems to [be](https://github.com/pagopa/interop-be-backend-for-frontend/blob/1.0.x/src/main/resources/application.conf#L27) `SUPPORT_LANDING_TOKEN_DURATION_SECONDS`. 

If I'm not right I may be confused on how to get env var from the BFF scala code, sorry about that.

Reply:
you are totally right, my bad! 😵‍💫